src/components/ArticleForm.jsx

```diff
@@ -2,24 +2,36 @@ import React, { useState } from "react";
 import { Form, Container, Checkbox, Segment, Message } from "semantic-ui-react";
 import Article from "../modules/articles";
 import { useHistory } from "react-router-dom";
+import toBase64 from "../modules/toBase64";
 
 const ArticleForm = () => {
   const [message, setMessage] = useState("");
   const [selectedCategory, setSelectedCategory] = useState("");
+  const [image, setImage] = useState();
   const history = useHistory();
 
+  const selectImage = (e) => {
+    setImage(e.target.files[0]);
+  };
+
   const handleCategoryChange = (value) => {
     setSelectedCategory(value);
   };
 
   const onSubmit = async (e) => {
     e.preventDefault();
+    let { title, teaser, content, premium, encodedImage } = e.target;
```

Comment:
move encodedImage out of here, that should be its own `let`.
So just do `let encodedImage` on the line bellow

Reply:
Moved it out!